From 6f9a22899c6734bde934b2e483fe061e0d61ce14 Mon Sep 17 00:00:00 2001
From: Peter Yoon <peyo.hd@gmail.com>
Date: Thu, 3 Aug 2017 09:29:52 -0400
Subject: [PATCH 1/2] rpi3: Enable GLSurfaceView

Currently mesa3d vc4 build is supporting only following formats.

- RGB 565, Alpha 0, Depth 24 or 0, Stencil 8 or 0
- RGB 888, Alpha 8, Depth 24 or 0, Stencil 8 or 0

Change-Id: Id00ea1df8846e6d7dd36a7915fd1d51138364283
---
 opengl/java/android/opengl/GLSurfaceView.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/opengl/java/android/opengl/GLSurfaceView.java b/opengl/java/android/opengl/GLSurfaceView.java
index 0f5dd3a..08d31c1 100644
--- a/opengl/java/android/opengl/GLSurfaceView.java
+++ b/opengl/java/android/opengl/GLSurfaceView.java
@@ -992,7 +992,7 @@ public class GLSurfaceView extends SurfaceView implements SurfaceHolder.Callback
      */
     private class SimpleEGLConfigChooser extends ComponentSizeChooser {
         public SimpleEGLConfigChooser(boolean withDepthBuffer) {
-            super(8, 8, 8, 0, withDepthBuffer ? 16 : 0, 0);
+            super(8, 8, 8, 8, withDepthBuffer ? 24 : 0, 0);
         }
     }
 
-- 
2.7.4

